script/apps/Aries/Creator/Game/Login/MainLogin.lua

```diff
@@ -547,7 +551,7 @@ function MainLogin:CheckShowTouchVirtualKeyboard()
 	if(System.options.IsTouchDevice) then
 		NPL.load("(gl)script/apps/Aries/Creator/Game/GUI/TouchVirtualKeyboardIcon.lua");
 		local TouchVirtualKeyboardIcon = commonlib.gettable("MyCompany.Aries.Game.GUI.TouchVirtualKeyboardIcon");
-		TouchVirtualKeyboardIcon.ShowSingleton(true);
+		TouchVirtualKeyboardIcon.ShowSingleton(false);
```

Comment:
Are you disabling this? Paracraft needs it to be shown by default. Maybe you can hide it in your logitow mod code.